oauth2_http/java/com/google/auth/oauth2/CloudShellCredentials.java

```diff
@@ -145,7 +146,7 @@ public int getAuthPort() {
 
     @Override
     public CloudShellCredentials build() {
-      return new CloudShellCredentials(authPort);
+      return new CloudShellCredentials(this);
```

Comment:
Can you please add a test  for this, similar to toBuilder_build in ComputeCredentialsTests?